oaknational/Oak-Web-Application
# fix: error pages on old safari devices hasOwn LESQ-776

## Description

Music year: 2001

https://caniuse.com/mdn-javascript_builtins_object_hasown
- hasOwn is not supported on older IOS versions causing them to crash , this swaps it out for prototype hasOwnProperty



## How to test

1. Go to https://deploy-preview-2361--oak-web-application.netlify.thenational.academy
Check links from slack in in IOS versions below 15.3

https://oaknationalacademy.slack.com/archives/C03HZ76AE3Z/p1713253441029169


## Screenshots


## Checklist
